src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -215,6 +215,7 @@ public function createTable(Table $table)
             $column = new Column();
             $column->setName('id')
                    ->setType('integer')
+                   ->setSigned(isset($options['signed']) ? $options['signed'] : false)
```

Comment:
> The way to do it without breaking BC would be to keep the current behavior (signed by default) and respect the signed option to create unsigned PKs.

I thought about it and that was a second option... On the one hand it breaks a backwards compatibility but on the other hand is `signed` pk is right behavior or we just want to save backwards compatibility forever? We can inform users via changelog about new pk behavior as it was done with PHP 5.3 support, by the way nobody forbids to download an older version of Phinx.

Also we may add new property to phinx.yml like `signed_pk: false` or something.

Let's discuss.